javascript/core/functions-i/access-a-function-s-arguments-as-an-array.md

```diff
@@ -23,7 +23,9 @@ aspects:
 ---
 ## Content
 
-The [arguments](https://enki.com/glossary/general/parameter-vs-argument.md) of a function can be accessed with the `arguments` keyword, but `arguments` only returns a *pseudo-array*.
+The [arguments](https://enki.com/glossary/general/parameter-vs-argument.md) of a function can be accessed with the `arguments` keyword.
+
+Do note that the value in `arguments` is a *pseudo-array* (an object with a length) that doesn't have all of the functionality of JavaScript arrays.
```

Comment:
Since the next sentence talks about `Array`, let's actually maybe mention that?

```suggestion
Do note that the value in `arguments` is a *pseudo-array* (an object with a length) that doesn't have all of the functionality of the JavaScript `Array` object.
```